tests/test_modeling_tf_bart.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2020 The Huggingface Inc. team
+# Copyright 2021 The Fairseq Authors and The HuggingFace Inc. team. All rights reserved.
```

Comment:
As Sylvain likes to ask, did the Fairseq authors really contribute to this test file?

Reply:
You're stealing my lines ;-)